data/json/items/generic.json

```diff
@@ -1383,6 +1383,20 @@
     "volume": "250 ml",
     "to_hit": -1
   },
+  {
+    "type": "GENERIC",
+    "id": "large_lcd_screen",
+    "symbol": ",",
+    "color": "dark_gray",
+    "name": "large LCD screen",
+    "category": "spare_parts",
+    "description": "A large backlit screen, used for displaying images.  Useful in some electronics recipes.",
+    "price": 150000,
```

Comment:
Please add a post-apocalypse price - I doubt these screens would cost $1500 after the Cataclysm.

Reply:
[Price was far too high and has been changed to $100.](https://github.com/CleverRaven/Cataclysm-DDA/pull/35320/commits/33b0bbd29151b7523ab8a9f79db91456a4c939c8)